gubled/gubled.go

```diff
@@ -26,7 +26,7 @@ type Args struct {
 	GcmApiKey   string `arg:"--gcm-api-key: The Google API Key for Google Cloud Messaging" env:"GUBLE_GCM_API_KEY"`
 }
 
-var ValidateStoragePath = func(args Args) error {
+func ValidateStoragePath(args Args) error {
```

Comment:
Having this creator/factory-functions as global, public variables was intended:
This gives a user of guble the possibility, to write its own main package, 
overwrite one or two of this functions and start guble with customized implementations of those modules.